app/db/skeema/bookmarkEntries.sql

```diff
@@ -11,5 +11,6 @@ CREATE TABLE `bookmarkEntries` (
   PRIMARY KEY (`id`),
   KEY `bookmarkEntries_userId_createdAt_key` (`userId`,`createdAt`),
   KEY `bookmarkEntries_videoId_key` (`videoId`),
-  KEY `bookmarkEntries_captionEntryId_key` (`captionEntryId`)
+  KEY `bookmarkEntries_captionEntryId_key` (`captionEntryId`),
+  FULLTEXT KEY `text` (`text`) /*!50100 WITH PARSER `ngram` */ 
```

Comment:
skeema put extra trailing space after magic comment.
It's a bit too annoying. Not sure if there's any workaround.